sherpa/utils/tests/test_logging.py

```diff
@@ -0,0 +1,37 @@
+#
+#  Copyright (C) 2020  Smithsonian Astrophysical Observatory
+#
+#
+#  This program is free software; you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation; either version 3 of the License, or
+#  (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License along
+#  with this program; if not, write to the Free Software Foundation, Inc.,
+#  51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+import pytest
+import logging
+import numpy as np
+import sherpa
+from sherpa.utils.logging import config_logger, SherpaVerbosity
+from sherpa.astro import ui
+
+logger = logging.getLogger('sherpa.some_module')
```

Comment:
A really pedantic point, but I think this should be declared within `test_logging_blah_blah` as there's no need for it to be a package global (and the fact we use a "random" name for the logger could potentlally-but-incredibly-unlikely cause confusion if we ever add more tests to this file).